webapp/pages/settings/notifications.spec.js

```diff
@@ -47,22 +47,19 @@ describe('notifications.vue', () => {
     })
 
     it('renders', () => {
-      expect(wrapper.is('.base-card')).toBe(true)
+      expect(wrapper.classes('base-card')).toBe(true)
     })
 
     it('clicking on submit changes notifyByEmail to false', async () => {
-      wrapper.find('#send-email').trigger('click')
-      await wrapper.vm.$nextTick()
-      wrapper.find('.base-button').trigger('click')
+      await wrapper.find('#send-email').setChecked(false)
```

Comment:
same x2

Reply:
This is not working anymore. You can click buttons, but you cannot click check boxes and radio buttions